var/spack/repos/builtin.mock/packages/test_import_resources/resources.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
It would be nice to have tests for import_resources with 0, 1, and 2+ resources. You have 2+ taken care of, but you don't test 0 or 1.

Reply:
will do

Reply:
done.